Makefile

```diff
@@ -24,7 +24,7 @@ manifests: controller-gen
 
 .PHONY: test
 test: fmt vet ## Run tests.
-	ginkgo --race -p --github-output -coverprofile cover.out -covermode atomic --json-report=test-report.json -v ././cmd/... ././pkg/...
+	ginkgo --race -p --procs=4 --randomize-suites --flake-attempts=3 --github-output -coverprofile cover.out -covermode atomic --json-report=test-report.json -v ././cmd/... ././pkg/...
```

Comment:
I'm not sure `--flake-attemps=3` is the way to go. It could hide other problems. If they are not extremely unrelyable, I'd suggest trying to fix the unit tests first.

https://github.com/konflux-ci/multi-platform-controller/pull/428 should help stabilize them.